src/commands/notify.yml

```diff
@@ -8,6 +8,10 @@ parameters:
     description: "IRC server port. Defaults to 6667."
     type: integer
     default: 6667
+  pass:
+    description: "The IRC server password."
+    type: string
+    default: ""
```

Comment:
I'm tempted to say this should be an environment variable rather than a plain string, so it's redacted in build logs *just in case*.

Reply:
Good point. Updated it to use an env variable like `server` does.